### PR TITLE
#10385: Migrate Unary forward ops to TTNN

### LINKED ISSUE #10385: Migrate 3 unary composite ops to TTNN
Unary Composite Tracking Issue : #9911  [Master issue : #9322  ]
List of ops required to be migrated :

- [x] Rpow
- [x] Normalize_global
- [x]  frac

### PR DESCRIPTION
**Pull request by mouliraj-mcw**
Issue : https://github.com/tenstorrent/tt-metal/issues/10385

Work Done:

Migrated 3 Unary ops to TTNN.
List of Ops:

- [x] rpow
- [x] Normalize_global
- [x]  frac

##Checklist
- [x] Post commit CI https://github.com/tenstorrent/tt-metal/actions/runs/10112439857
- [ ] [post-commit] models tests 
- [ ] Nightly fast dispatch- 
- [x] New/Existing tests provide coverage for changes

### CODE REVIEW

File: tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py [hunk 1]
```diff
@@ -2562,7 +2572,7 @@ def unary_op_optional_output_with_fast_approx(
 # mean_global = make_unary_op(ttl.tensor.global_mean)
 # var_global = make_unary_op(ttl.tensor.global_var)
 # std_global = make_unary_op(ttl.tensor.global_std)
-normalize_global = make_unary_op(ttl.tensor.normalize_global)
+# normalize_global = make_unary_op(ttl.tensor.normalize_global)
```
Comment (umadevimcw) [inline tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py:2575]: Remove the commented code

**Review by umadevimcw — CHANGES_REQUESTED** [review]

File: tests/tt_eager/python_api_testing/non_working_unit_tests/wormhole/test_normalize_global.py [hunk 2]
```diff
@@ -72,7 +72,7 @@ def run_normalize_global_test(input_shape, dtype, dlayout, in_mem_config, out_me
         14854324,
     ),
     (
-        (4, 12, 32, 104),
+        (4, 12, 32, 128),
         [ttl.tensor.DataType.BFLOAT16],
         [ttl.tensor.Layout.ROW_MAJOR],
         [
```
Comment (VirdhatchaniKN) [inline tests/tt_eager/python_api_testing/non_working_unit_tests/wormhole/test_normalize_global.py:78]: Please implement golden function for this op

File: ttnn/cpp/ttnn/operations/eltwise/unary/device/unary_composite_op.cpp [hunk 3]
```diff
@@ -610,4 +610,37 @@ Tensor _glu(const Tensor& input_a, int32_t dim , const std::optional<MemoryConfi
...
+
+    // reformat
+    Tensor z_1 = reshape(z_0, y.get_legacy_shape()[0], y.get_legacy_shape()[1], y.get_legacy_shape()[2], y.get_legacy_shape()[3]);
+    z_0.deallocate();
+
+    return z_1;
+}
+
+// Global Norm
+Tensor _normalize_global(const Tensor& y,  const std::optional<MemoryConfig>& output_mem_config) {
```
Comment (KalaivaniMCW) [inline ttnn/cpp/ttnn/operations/eltwise/unary/device/unary_composite_op.cpp:642]: https://github.com/tenstorrent/tt-metal/blob/953d2c504d9f1d4f5917888f422e10f13f5377c1/tests/tt_eager/python_api_testing/sweep_tests/pytorch_ops.py#L83

https://github.com/tenstorrent/tt-metal/blob/953d2c504d9f1d4f5917888f422e10f13f5377c1/tests/tt_eager/python_api_testing/sweep_tests/pytorch_ops.py#L1447

attach golden function for both ops in unary.py

Comment (KalaivaniMCW) [reply ttnn/cpp/ttnn/operations/eltwise/unary/device/unary_composite_op.cpp:642]: use above links for golden fn  reference

**Review by umadevimcw — APPROVED** [review]

**Review by eyonland — APPROVED** [review]

**Pull request merged**